test/junit/src/test/scala/strawman/collection/mutable/LinkedHashMapTest.scala

```diff
@@ -1,15 +1,13 @@
-package scala.collection.mutable
+package strawman.collection.mutable
```

Comment:
I enabled these tests but they cover almost nothing. I’ve had a look at the scalacheck tests but they don’t seem to cover mutable map either…